curator/static/js/tnrs/tnrs-main.js

```diff
@@ -116,47 +117,51 @@ var getNewNamesetModel = function(options) {
     */
     return obj;
 };
+
+var papaParseConfig = {
+    /* Try to accomodate different delimiters, line endings, etc.
+     * For all settings and default values, see <https://www.papaparse.com/docs#config>
+     */
+    delimiter: "",	// auto-detect
+    delimitersToGuess: [', ', ',', '\t', '|', ';', Papa.RECORD_SEP, Papa.UNIT_SEP],
```

Comment:
Sadly, suggesting ', ' as a possible delimiter didn't fix our leading-space problem. It seems to be ignored or rejected by Papa Parse, though [their code seems to accept multi-character delimiters](https://github.com/mholt/PapaParse/blob/e11ee26581d9f497087b4c3ee00c2a77645b81d2/papaparse.js#L1294). Odd.